rbvmomi2.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency('builder', '~> 3.2')
   spec.add_runtime_dependency('json', '~> 2.3')
   spec.add_runtime_dependency('nokogiri', '>= 1.12.5', '~> 1.12')
+  spec.add_runtime_dependency('mini_portile2')
```

Comment:
@Fryguy not sure if you know why but on windows bundle install fails unless I add mini_portile2 as an explicit dependency

Reply:
I've seen that elsewhere, but I thought bundler handled that automatically.  Wonder if there's a bundler config somewhere that's forcing Ruby instead of windows?

Reply:
Maybe just bump to nokogiri 1.13?

Reply:
Yeah so this was a run that failed https://github.com/ManageIQ/rbvmomi2/actions/runs/3229877984/jobs/5287671947#step:3:319
I get the `Your lockfile was created by an old Bundler that left some things out.` warning even with the latest bundler, rubygems, and no cached Gemfile.lock so I don't think it is an old Bundler and then the eventual failure is https://github.com/ManageIQ/rbvmomi2/actions/runs/3229877984/jobs/5287671947#step:3:353

Reply:
It was pulling 1.13.8 (https://github.com/ManageIQ/rbvmomi2/actions/runs/3229877984/jobs/5287671947#step:3:348) but I can try bumping the version anyway

Reply:
Yeah same failure https://github.com/ManageIQ/rbvmomi2/actions/runs/3230226743/jobs/5288431075#step:3:73

Reply:
Only other thing I can think of is maybe they broke 1.13.8 when they built it...maybe try 1.13.6 or 5, but if that still doesn't work, then we can just go with this workaround.